ansible_collections/arista/cvp/plugins/module_utils/device_tools.py

```diff
@@ -1699,11 +1699,12 @@ def decommission_device(self, user_inventory: DeviceInventory):
                 status = ""
                 while status != decomm_success:
```

Comment:
@noredistribution, I think this should be an "if" condition to avoid infinite loop on "status == decomm_failure"